scripts/data_files/vs2010-main-template.vcxproj

```diff
@@ -107,6 +110,12 @@
     <Link>
       <SubSystem>Windows</SubSystem>
       <GenerateDebugInformation>true</GenerateDebugInformation>
+<<<<<<< HEAD:scripts/data_files/vs2010-main-template.vcxproj
+=======
+      <ShowProgress>NotSet</ShowProgress>
+      <AdditionalDependencies>bcrypt.lib;kernel32.lib;user32.lib;gdi32.lib;winspool.lib;comdlg32.lib;advapi32.lib;shell32.lib;ole32.lib;oleaut32.lib;uuid.lib;odbc32.lib;odbccp32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalLibraryDirectories>Debug</AdditionalLibraryDirectories>
+>>>>>>> Replace Windows APIs that are banned in Windows Store apps:visualc/VS2010/md5sum.vcxproj

```

Comment:
Probably not a good idea to have conflict markers here...

Reply:
Ooops! Sorry. Now fixed.